### PR TITLE
Minor: Add 'no_password' field in createUser / updateUser. RD-20955

### PR DESCRIPTION
**Pull request by alexeykondratenko**

### CODE REVIEW

File: specs/engage-digital_postman2.json [hunk 1]
```diff
@@ -5387,6 +5387,12 @@
                       "value": "\u003carray.string.csv\u003e",
                       "description": "List of locales corresponding to the languages spoken by the user (multiple).",
                       "disabled": true
+                    },
+                    {
+                      "key": "no_password",
+                      "value": "\u003cboolean\u003e",
+                      "description": "Whether the user is passwordless or not (boolean).  Must be used with the corresponding SSO configuration.",
```
Comment (ArtyomIvanov-dev) [inline specs/engage-digital_postman2.json:5394]: Do we need an extra space here?

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -6028,6 +6036,14 @@ paths:
             items:
               type: string
             type: array
+        - description: >-
+            Whether the user is passwordless or not (boolean).
```
Comment (ArtyomIvanov-dev) [inline specs/engage-digital_openapi3.yaml:6040]: Nitpick: We don't need 'boolean' here because this information is in the schema. 

Up to you

**Comment by ArtyomIvanov-dev** [comment]
LGTM

File: specs/engage-digital_postman2.json (hunk 1 above)
Comment (alexeykondratenko) [reply specs/engage-digital_postman2.json]: Removed. Thanks.

File: specs/engage-digital_openapi3.yaml (hunk 2 above)
Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: I copied from other properties, I think we can keep the same approach here.

**Pull request merged**